src/utils/timeUtils.ts

```diff
@@ -0,0 +1,33 @@
+// Generate hourly time options array used for EarliestTime and LatestTime dropdowns
```

Comment:
I'd have suggested make dateUtils into datetimeUtils instead of two files but this is fine for now if you see this growing separately.

Reply:
I might need to add logic to sort days of the week after user selects/deselects days, so will keep these separate for now.